src/main/java/crawlercommons/domains/EffectiveTldFinder.java

```diff
@@ -65,26 +96,73 @@
      * A singleton
      */
     private EffectiveTldFinder() {
-        initialize(null);
+        initialize(this.getClass().getResourceAsStream(ETLD_DATA));
     }
 
+    /**
+     * Get singleton instance of EffectiveTldFinder with default configuration.
+     * 
+     * @return singleton instance of EffectiveTldFinder
+     */
     public static EffectiveTldFinder getInstance() {
         if (null == instance) {
             instance = new EffectiveTldFinder();
         }
         return instance;
     }
 
+    /**
+     * (Re)initialize EffectiveTldFinder with built-in public suffix list.
+     * 
+     * @param excludePrivateDomains
+     *            whether to exclude the public suffixes listed in the PRIVATE
+     *            domain section (opposed to &quot;ICANN&quot; domains)
+     * @return true if (re)initialization was successful
+     */
+    public boolean initialize(boolean excludePrivateDomains) {
+        return initialize(this.getClass().getResourceAsStream(ETLD_DATA), false);
```

Comment:
Shouldn't this pass `excludePrivateDomains` to the `initialize()` call, versus always passing `false`?

Reply:
Sure, I'll update the PR.